pytac/load_csv.py

```diff
@@ -1,3 +1,5 @@
+""" Class to load the elements of the machine from a csv file."""
+
```

Comment:
The module can contain multiple routines or different classes. My bad for calling it a class instead of a module.